src/widget/whotcuebutton.cpp

```diff
@@ -130,6 +141,42 @@ void WHotcueButton::slotColorChanged(double color) {
     restyleAndRepaint();
 }
 
+void WHotcueButton::slotTypeChanged(double type) {
+    switch (static_cast<mixxx::CueType>(type)) {
```

Comment:
CI complains about this:

```
/home/travis/build/mixxxdj/mixxx/src/widget/whotcuebutton.cpp: In member function ‘void WHotcueButton::slotTypeChanged(double)’:
/home/travis/build/mixxxdj/mixxx/src/widget/whotcuebutton.cpp:144:44: error: parameter ‘type’ set but not used [-Werror=unused-but-set-parameter]
 void WHotcueButton::slotTypeChanged(double type) {
                                            ^~~~
cc1plus: all warnings being treated as errors
```

I don't get it. It's used in the switch case in line 145. Maybe a compiler bug?

Reply:
- Try to assign to a local (const) variable `cueType`
- Try to cast to int (default underlying type) before casting to the enum class

Reply:
...and finally leave a comment ;)

Reply:
https://github.com/mixxxdj/mixxx/pull/2194/commits/8597162f0fb1c45c6abbfc517fe1cc3145638b50

Let's see what the CI says to that.

Reply:
Didn't work, I'll try to cast it to int first.

Reply:
That seems to have fixed it.